content/purgatorio/collect-and-prepare-data/data-collection.md

```diff
@@ -0,0 +1,209 @@
+---
+title: Data Collection
+author: neomatrix369
+description: The purpose of this guide is to talk about data collection and the different steps you need to do to prepare your dataset even before we can think of Machine Learning models and the works.
+---
+
+# What you will learn 
+The real world is full of data, there are many types of data and collecting them is an art and comes through experience and doing it in an iterative process.
+
+Also this is one of the preliminary steps when you are about to do Machine Learning or Deep Learning or any of the AI things. The cascading steps or processes depend quite a bit on this task but at the same time, this is a task that can be iterated a number of times.
```

Comment:
I would change "AI things" in something less generic. I'll say:
... This is the first and preliminary step when it comes to applying some sort of science to data, or building Machine Learning or Deep Learning Models.

Reply:
Done!